package.json

```diff
@@ -1,16 +1,16 @@
 {
   "name": "istanbul-instrumenter-loader",
-  "version": "3.0.1",
+  "version": "3.0.2",
```

Comment:
Don't do this, because we use standard-version for updating version

Reply:
I'll revert that